### PR TITLE
Adding tool binaries to PATH

### PR DESCRIPTION
**Pull request by shigupt202**
Fixes https://github.com/Azure/setup-kubectl/issues/3

### CODE REVIEW

File: src/run.ts [hunk 1]
```diff
@@ -70,6 +70,17 @@ async function run() {
         version = await getStableKubectlVersion();
     }
     let cachedPath = await downloadKubectl(version);
+
+    try {
+
+        if (!process.env['PATH'].startsWith(path.dirname(cachedPath))) {
```
Comment (ds-ms) [inline src/run.ts:76]: You can skip the startsWith and add it by default maybe? 

Cause the order might differ based on the order other tools are installed?

File: src/run.ts [hunk 2]
```diff
@@ -70,6 +70,17 @@ async function run() {
...
     }
     let cachedPath = await downloadKubectl(version);
+
+    try {
+
+        if (!process.env['PATH'].startsWith(path.dirname(cachedPath))) {
+            core.addPath(path.dirname(cachedPath));
+        }
+    }
+    catch {
```
Comment (ds-ms) [inline src/run.ts:80]: What is expected to throw an error here?

File: src/run.ts (hunk 1 above)
Comment (shigupt202) [reply src/run.ts:76]: Ok

File: src/run.ts (hunk 2 above)
Comment (shigupt202) [reply src/run.ts:80]: The if condition could throw an error. But since we're removing that, we can remove the try catch also.

**Pull request merged**